Add Trendline option for focalplane and positioner panels

apaat opened the pull request:
-Added a trendline option for the focalplane and positioner panels.
-Trendline will appear for all three plots (correlation plot and two datetime plots)
-Trendline will plot raw data unless 'binned data' is selected

apaat committed:
Add Trendline option for focalplane and positioner panels

-Added a trendline option for the focalplane and positioner panels.
-Trendline will appear for all three plots (correlation plot and two datetime plots)

parfa30 commented:
Anthony, looks great! Could you change the line from many dots to a clean straight line? Also, I think if we could print the line attributes (slope, y-int) on the plot that would be good to.

apaat committed:
Added slope and y-intercept values to page

-Added slope and y-intercept values to page, these will show up as 'NA' if plot trend line is not selected
-Changed line to be thinner and changed the color to black

parfa30 committed:
changed location of tredline widgets

parfa30 committed:
Merge branch 'master' into paat_trendline
